esp/src/test-ui/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd ./tests
```

Comment:
I think this should be removed, not assume in script to be where tests are,
and instead the current dir used.
This is probably related to the copying of the tests to ./tests, which could be avoided.

Reply:
Removed and reworked.